Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = [".", "testing/e2e", "identity", "testing/itest"]
+members = [".", "testing/e2e", "identity", "testing/itest", "eth-keystore"]
```

Comment:
yeah, initially I was thinking along this line, but when I started to implement it, I realised it would make the `identity` crate quite messy or need some refactoring, like renaming some struct and move around the files. So I started with a new crate.

Reply:
I see... I think it is worth doing the effort now of following that approach, it may be harder to refactor in the future, WDYT?

Reply:
this works, time to see how good the IDE is.